lxd/instance/drivers/driver_qemu.go

```diff
@@ -300,6 +300,16 @@ func qemuCreate(s *state.State, args db.InstanceArgs, p api.Project) (instance.I
 		return nil, nil, fmt.Errorf("Failed loading storage pool: %w", err)
 	}
 
+	if d.storagePool.Driver().Info().Remote {
+		if d.enableMigrationStateful() {
```

Comment:
This isn't what we agreed to do in our Cloud team meeting as far as I recall.

We agreed to make instances.migration.stateful apply dynamically at start time by default if not overridden in the profile and instance configs. Rather than write a config entry into the new instance DB record.

Reply:
Ah sorry for the misunderstanding. I'm fixing that.